src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_skadi.cpp

```diff
@@ -160,88 +147,52 @@ class boss_skadi : public CreatureScript
 public:
     boss_skadi() : CreatureScript("boss_skadi") { }
 
-    CreatureAI* GetAI(Creature* creature) const override
-    {
-        return GetInstanceAI<boss_skadiAI>(creature);
-    }
-
     struct boss_skadiAI : public ScriptedAI
     {
         boss_skadiAI(Creature* creature) : ScriptedAI(creature), Summons(me)
         {
-            Initialize();
             instance = creature->GetInstanceScript();
-            m_uiMovementTimer = 0;
-            m_uiSummonTimer = 0;
-        }
-
-        void Initialize()
```

Comment:
why did you remove Initialize() method ? Now the constructor is not initializing all class fields